tests/behaviour/steps/cli.py

```diff
@@ -7,6 +12,9 @@
 from appdirs import AppDirs
 from behave import then, when
 from csvcubeddevtools.behaviour.temporarydirectory import get_context_temp_dir_path
+from csvcubeddevtools.helpers.file import get_test_cases_dir
+
+_test_case_dir = get_test_cases_dir()
```

Comment:
I don't think this chunk is necessary in here anymore.

Reply:
Yeah that is true I took it out